employee searchform

karimovich86 opened the pull request:
[User Story 1](https://github.com/karimovich86/capstone-project/issues/1)
[User Story 2](https://github.com/karimovich86/capstone-project/issues/2)


accidentely skiped first PR. this one is for both User stories

karimovich86 committed:
created Employees List

karimovich86 committed:
implent searchform

karimovich86 committed:
vercel test

MerleMak commented on components/Fomular/index.js:
```diff
@@ -1 +1,41 @@
+import { useState } from "react";
+import { employees } from "@/test_daten/test_daten";
+
+function Formular() {
```
```suggestion
function SearchForm() {
```

MerleMak commented on components/Fomular/index.js:
```diff
@@ -1 +1,41 @@
+  function handleShowAll() {
+    setSearchValue("");
+  }
+  // vercel test
```
```suggestion

```

MerleMak commented on components/Fomular/index.js:
```diff
@@ -1 +1,41 @@
+  );
+}
+
+export default Formular;
```
```suggestion
export default SearchForm;
```

MerleMak commented on pages/index.js:
```diff
@@ -1,3 +1,9 @@
+  return (
+    <>
+      <h1>HR Manager App</h1>
+      <Formular />
```
```suggestion
import SearchForm from "@/components/SearchForm.js";
export default function Homepage() {
  return (
    <>
      <h1>HR Manager App</h1>
      <SearchForm />
```

MerleMak commented on package.json:
```diff
@@ -12,10 +12,12 @@
   "dependencies": {
     "eslint": "8.39.0",
     "eslint-config-next": "13.3.1",
+    "lodash": "^4.17.21",
```
Es sieht so aus als würdest du dieses package nicht nutzen, es kann gelöscht werden

MerleMak commented on package.json:
```diff
@@ -12,10 +12,12 @@
     "react-dom": "18.2.0",
-    "styled-components": "^5.3.10"
+    "styled-components": "^5.3.10",
+    "uuid": "^9.0.0"
```
genau wie dieses package. Bitte installiere die packages erst dann wenn du sie wirklich verwendest.

MerleMak reviewed (commented):
Hallo Karim, ich habe dir ein paar Kommentare hinterlassen. Bitte benenne deine Komponente um und installiere nur die packages die du brauchst :)

karimovich86 committed:
Update pages/index.js

error - ./pages/index.js:1:0
Module not found: Can't resolve '@/components/SearchForm.js'
> 1 | import SearchForm from "@/components/SearchForm.js";
  2 | export default function Homepage() {
  3 |   return (
  4 |     <>

Co-authored-by: merlemak <merlemakoben@gmail.com>

karimovich86 committed:
corrections PR Done

karimovich86 committed:
git fix

karimovich86 committed:
merge conflict

karimovich86 committed:
vercel fix bug

MerleMak reviewed (approved):
Sieht gut aus!
